esphome/components/ld2410s/ld2410s.h

```diff
@@ -0,0 +1,207 @@
+#pragma once
+// #define LD2410S_V2
+#include "esphome/core/application.h"
+#include "esphome/core/automation.h"
+#include "esphome/core/component.h"
+#include "esphome/core/defines.h"
+#include "esphome/core/helpers.h"
+#include "esphome/core/log.h"
+#include "esphome/components/uart/uart.h"
+#ifdef LD2410S_V2
+#endif
+#include <functional>
+#include <iomanip>
+#include <cstddef>
+#include <cstdint>
+#include <type_traits>
+namespace esphome {
+namespace ld2410s {
```

Comment:
```suggestion
namespace esphome::ld2410s {
```

c++17 namespaces for new components please